dist/tools/openocd/openocd.sh

```diff
@@ -202,7 +202,7 @@ do_debug() {
             -c 'halt' \
             -l /dev/null" &
     # save PID for terminating the server afterwards
-    OCD_PID=$?
+    OCD_PID=$(($!+2))
```

Comment:
the +2 feels kind of arbitrary. Is it guaranteed to be the same every time on every system?